Sources/Models/Resources/NamedColor.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  NamedColor.swift
+//  Tests
+//
+//  Created by phimage on 01/04/2018.
+//
+
+import SWXMLHash
+
+public struct NamedColor: XMLDecodable, ResourceProtocol {
```

Comment:
Can you link me to something that has this XML element? I've never seen it before